### PR TITLE
rocky8 kvm docker steps

### PR DESCRIPTION
**Pull request by rajujith**
updated the docker installation steps for rocky8

### CODE REVIEW

File: playbooks/roles/docker/tasks/RedHat.yml [hunk 1]
```diff
@@ -16,28 +16,9 @@
     gpgcheck: no
   when: docker_mirror_url is defined
 
-# docker-ce-18 requries pigz which is available starting from rhel8
-# so, use epel
-- name: configure epel
-  yum:
-    name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ ansible_distribution_major_version }}.noarch.rpm
-  when: ansible_distribution_major_version|int < 8
```
Comment (weizhouapache) [inline playbooks/roles/docker/tasks/RedHat.yml:24]: @rajujith 
I believe this works in rocky8. will deployment on centos7 be broken if these steps are removed ?

File: playbooks/roles/docker/tasks/RedHat.yml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-    gpgcheck: yes
-  when: docker_mirror_url is not defined
+- name: Install yum-utils
+  yum:
+     name: yum-utils
+     state: present
+
+- name: Add Docker repo
+  command:
+     cmd: yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
```
Comment (weizhouapache) [inline playbooks/roles/docker/tasks/RedHat.yml:9]: @rajujith 
are these supported by centos7 ?

Comment (rajujith) [reply playbooks/roles/docker/tasks/RedHat.yml:9]: yes

File: playbooks/roles/docker/tasks/RedHat.yml (hunk 1 above)
Comment (rajujith) [reply playbooks/roles/docker/tasks/RedHat.yml:24]: It should still work.

**Review by weizhouapache — APPROVED** [review]

**Pull request merged**